packages/eventsub-http/src/EventSubHttpBase.ts

```diff
@@ -211,16 +211,17 @@ export abstract class EventSubHttpBase extends EventSubBase {
 			} else if (type === 'notification') {
 				if (new Date(timestamp).getTime() < Date.now() - 10 * 60 * 1000) {
 					this._logger.debug(`Old notification(s) prevented for event: ${id}`);
-				} else {
-					const payload = data as EventSubNotificationPayload;
-					if ('events' in payload) {
-						for (const event of payload.events) {
-							this._handleSingleEventPayload(subscription, event.data, event.id);
-						}
-					} else {
-						this._handleSingleEventPayload(subscription, payload.event, messageId);
+				}
```

Comment:
This `else` was *not* unnecessary. Removing it renders the above check useless.

Reply:
Oops, you're right, reverting.